JavaScript/Update_README_With_Hashnode_Blogs/update_readme.js

```diff
@@ -0,0 +1,44 @@
+const fetch = require("node-fetch");
+const fs = require("fs");
+const query = `
+    {
+      user(username: "<your Hashnode username>") {
+        publication {
+          posts{
+            slug
+            title
+          }
+        }
+      }
+    }
+  `;
+
+const fetchPosts = async () => {
+  const result = await fetch("https://api.hashnode.com", {
+    method: "POST",
+    headers: {
+      "Content-type": "application/json",
+    },
+    body: JSON.stringify({ query }),
+  });
+  const response = await result.json();
+  const posts = response.data.user.publication.posts.slice(0, 3);
+  const postsToShow = `\n - [${posts[0].title}](<you hashnode blog URL>/${posts[0].slug})\n - [${posts[1].title}](<you hashnode blog URL>/${posts[1].slug})\n - [${posts[2].title}](<you hashnode blog URL>/${posts[2].slug})\n\n`;
+  fs.readFile("README.md", "utf-8", (err, data) => {
+    if (err) {
+      throw err;
+    }
+    console.log(postsToShow);
```

Comment:
Any specific reason to `console.log()`? If you are logging then make sure it is a readable log. Add some information along with logging `postsToShow`; can be helpful to debug.

Reply:
It's just for debugging in the **action** to see that posts received successfully or not. 
![screenshot](https://imgur.com/xKAuk0X)

Reply:
Okay, awesome